tests/refinement/pdf/test_pdf.py

```diff
@@ -3,13 +3,35 @@
 
 from pytest_regressions.data_regression import DataRegressionFixture
 
-from paperoni.refinement.pdf.pdf import analyse_pdf
+from paperoni.model.classes import PaperInfo
+from paperoni.refinement.fetch import fetch_all
 from tests.utils import check_papers
 
 
 def test_analyse_pdf(data_regression: DataRegressionFixture):
-    with patch("paperoni.config.config.data_path", Path(__file__).parent / "data"):
-        pinfo = analyse_pdf("openreview", "_3FyT_W1DW")
+    def _check_pdf(pinfo: PaperInfo):
+        return "pdf" in [k.split(":")[0] for k in pinfo.info["refined_by"]]
+
+    with (
+        patch("paperoni.config.config.data_path", Path(__file__).parent / "data"),
```

Comment:
You can use `gifnoc.overlay` for config modifications: `with gifnoc.overlay({"key.subkey": value}): ...`